pymc4/distributions/distribution.py

```diff
@@ -123,6 +123,24 @@ def sample_numpy(self, sample_shape=(), seed=None):
         """
         return self.sample(sample_shape, seed).numpy()
 
+    def test_sample(self, sample_shape=(), seed=None):
```

Comment:
can this have a more descriptive name? I assumed this was testing sample...
```suggestion
    def get_test_value(self, sample_shape=(), seed=None):
```

Reply:
I'll change it today